### PR TITLE
Refs #28643 -- Added LPad and RPad database functions.

### PR DESCRIPTION
**Pull request by felixxm**
https://code.djangoproject.com/ticket/28643

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
buildbot, test on oracle.

File: docs/ref/models/database-functions.txt [hunk 1]
```diff
@@ -800,6 +800,27 @@ Usage example::
...
+``LPad``
+--------
+
+.. class:: LPad(expression, length, fill_text=Value(' '), **extra)
+
+.. versionadded:: 2.1
+
+Returns the value of the given text field or expression padded on the left-side
+with a ``fill_text`` to a specified ``length``. The default ``fill_text`` is
+the single space.
```
Comment (timgraham) [inline docs/ref/models/database-functions.txt]: the single space -> a space

File: docs/ref/models/database-functions.txt [hunk 2]
```diff
@@ -800,6 +800,27 @@ Usage example::
...
 
+``LPad``
+--------
+
+.. class:: LPad(expression, length, fill_text=Value(' '), **extra)
+
+.. versionadded:: 2.1
+
+Returns the value of the given text field or expression padded on the left-side
+with a ``fill_text`` to a specified ``length``. The default ``fill_text`` is
```
Comment (timgraham) [inline docs/ref/models/database-functions.txt]: "padded on the left side with `fill_text` so that the resulting value is `length` characters long."

File: docs/ref/models/database-functions.txt [hunk 3]
```diff
@@ -800,6 +800,27 @@ Usage example::
...
     margaret smith
 
+``LPad``
+--------
+
+.. class:: LPad(expression, length, fill_text=Value(' '), **extra)
+
+.. versionadded:: 2.1
+
+Returns the value of the given text field or expression padded on the left-side
```
Comment (timgraham) [inline docs/ref/models/database-functions.txt]: No dash for "left side" / "right side".

**Comment by felixxm** [comment]
@timgraham Thanks for the review.

**Comment by timgraham** [comment]
In https://github.com/django/django/pull/9220, Mariusz and I agreed that developers can use, for example, `import LPad as LJust` if they prefer a different name.

**Comment by bobort** [comment]
Okay, but that is not consistent with how we handled Trim and giving it an alias of Strip (the Pythonic term).

**Comment by timgraham** [comment]
We didn't include those aliases. https://github.com/django/django/pull/9220#discussion_r174330991

**Comment by bobort** [comment]
Well, I stand corrected.  (The ticket mentions in the comments that it was done.)  It's hard to follow these conversations since they are scattered in so many different places.
